### DISCUSSION TITLE
Update Maven project error

### DISCUSSION THREAD

**Discussion by hongli-github** [eclipse-m2e/m2e-core#969] [Q&A]
Hello,

I am leaning eclipse with Maven project. When I updated the maven project in IDE got error message: No appropriate protocol, may be no appropriate cipher suite specified or protocols are deactivated.

It is ok if I use command line in windows CMD.

Thanks you very much for your help.

Hope

**Comment by mickaelistria** [comment]
What's the version of Java you're running your command-line with?
Eclipse IDE uses Java 17, and if the transfer or protocol get disabled by default in Java 17 (unsafe algorithm get disabled by default in newer Java releases), it can explain why it fails in the IDE; but not in command-line that may use another Java version.

**Reply by HL-G** [reply]
it is websphere 8.5 and only one java version java 8 installed.
